api/server.spec.js

```diff
@@ -0,0 +1,139 @@
+const request = require('supertest');
+const server = require('./server.js');
+const db = require('../data/dbConfig.js');
+
+describe('route handlers', () => {
+    beforeEach( () => {
+        return db.migrate.rollback()
+        .then( () => {
+            return db.migrate.latest()
+            .then( () => {
+                return db.seed.run()
+            })
+        })
+    })
+    
+    afterEach(async () => {
+        await db.migrate.rollback()
+    })
+    
+    // 3 Tests per endpoint
+    describe('get /', () => {
+        // The GET /games endpoint should return the list of games and HTTP status code 200.
+        it('http Status code 200', async () => {
+            const success = await request(server).get('/');
+            expect(success.status).toBe(200);
+        });
+
+        it('should send back a proper response', async () => {
+            const success = await request(server).get('/');
+            expect(success.body.length).toBe(3);
+        });
+        
+        // Write a test to make sure this endpoint always returns an array, even if there are no games stored. If there are no 
+        // games to return, the endpoint should return an empty array.
+        it('sends empty array when empty', async () => {
+            const empty = await request(server).get('/');
+            expect(empty.body).toEqual([]);
+        });
+    });
+
+    describe('get by ID', () => {
+        it('Success responds with 200', async () => {
+            const success = await request(server).get('/2');
+            expect(succes.status).toBe(200);
```

Comment:
Got a little typo here
```suggestion
            expect(success.status).toBe(200);
```